locales/es/translation.json

```diff
@@ -1,1327 +1,1328 @@
+    "Credential not found.": "No se encontró la credencial.",
+    "Credential type": "Tipo de credencial",
+    "Credentials": "Credenciales",
+    "Credentials are used for authentication when launching Jobs against machines, synchronizing with inventory sources, and importing project content from a version control system.": "Las credenciales se utilizan para la autenticación al lanzar tareas contra máquinas, sincronizar con fuentes de inventario e importar contenido del proyecto desde un sistema de control de versiones.",
+    "Credentials are utilized by {{product}} for authentication when launching Jobs against machines, synchronizing with inventory sources, and importing project content from a version control system.": "Las credenciales son utilizadas por {{product}} para la autenticación cuando se lanzan Tareas contra máquinas, se sincronizan con fuentes de inventario y se importa el contenido del proyecto desde un sistema de control de versiones.",
+    "Credentials are utilized by {{product}} for authentication when launching Jobs against machines, synchronizing with inventory sources, and importing project content from a version control system. You can grant users and teams the ability to use these credentials, without actually exposing the credential to the user. If you have a user move to a different team or leave the organization, you don’t have to re-key all of your systems just because that credential was available in {{product}}.": "Las credenciales son utilizadas por {{product}} para la autenticación cuando se lanzan Tareas contra máquinas, se sincronizan con fuentes de inventario y se importa el contenido del proyecto desde un sistema de control de versiones. Puede conceder a los usuarios y equipos la capacidad de utilizar estas credenciales, sin exponer realmente la credencial al usuario. Si un usuario se traslada a otro equipo o abandona la organización, no tendrá que volver a introducir la clave en todos los sistemas sólo porque esa credencial estaba disponible en {{product}}.",
+    "Current page savings": "Ahorros de la página actual",
+    "Custom virtual environment {virtualEnvironment} must be replaced by an execution environment. For more information about migrating to execution environments see <4>the documentation.</4>": "Debe reemplazar el entorno virtual personalizado {virtualEnvironment} por un entorno de ejecución. Para obtener más información sobre la migración a entornos de ejecución, consulte &lt;4>la documentación.&lt;/4>",
+    "Dashboard": "Panel de control",
+    "Debug": "Debug",
+    "Decision environment": "Entorno de decisión",
+    "Decision Environment": "Entorno de decisión",
+    "Decision environments": "Entornos de decisión",
+    "Decision Environments": "Entornos de decisión",
+    "Decision environments contain a rulebook image that dictate where the rulebooks will run.": "Los entornos de decisión contienen una imagen del reglamento que dicta dónde se ejecutarán los reglamentos.",
```

Comment:
I think both, english and spanish are incomplete. I think it should be 
```json
"Decision environments contain an ansible-rulebook image that dictate where the rulebooks will run.": "Los entornos de decisión contienen una imagen de ansible-rulebook que dicta dónde se ejecutarán los reglamentos.",
```

Reply:
This one, we'll need to change the marked string in the code.

Reply:
I updated the marked string and the Spanish translation.